### PR TITLE
ploigos refactor

### PR DESCRIPTION
**Pull request by itewk**
# Purpose
* rename TSSC to ploigos

## sub-purpose
* rename Dockerfile to Containerfile
* manual-publish.sh - add --no-cache for more consistent builds
* remove ploigos-tool-skopeo in favor of ploigos-tool-containers

### CODE REVIEW

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -918,146 +918,21 @@ jobs:
...
 
     runs-on: ubuntu-latest
 
     env:
-      IMAGE_CONTEXT: ./tssc-tool-skopeo
-      IMAGE_FILE: Dockerfile
-      IMAGE_NAME: tssc-tool-skopeo
-      IMAGE_TAG_LOCAL: localhost:5000/${{ secrets.REGISTRY_REPOSITORY }}/tssc-tool-skopeo:latest
-      FROM_IMAGE_NAME: tssc-base
-
```
Comment (kfrankli) [inline .github/workflows/publish.yaml:936]: Is striking all the below intentional?

Comment (itewk) [reply .github/workflows/publish.yaml:936]: yes. we are getting rid of the skopeo image since the containers image has skopeo in it

**Review by kfrankli — APPROVED** [review]

File: ploigos-tool-containers/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# ploigos-tool-containers
+
+This repository contains the container definition for creating the Ploigos workflow
+container CLI tools container image.
+
+This container image is intended to be used as the container image to run Ploigos workflow steps
+in that require access to CLI tools related to container image operations.
+
+Includedm CLI tools:
```
[the remark this answers is not in the document]
Comment (itewk) [reply ploigos-tool-containers/README.md]: fixed

**Review by kfrankli — APPROVED** [review]

**Review by kharyam — APPROVED** [review]
Looks good!

**Pull request merged**